operator/pkg/base/base.go

```diff
@@ -18,6 +18,10 @@ type (
 		ImageRegistryURL string
 		ImagePullPolicy  string
 		SidecarImageName string
+		// The image name of the easeagent initializer
+		EaseagentInitializerImageName string
+		// Log4jConfigName default is easeagent-log4j.xml
```

Comment:
```suggestion
		// Log4jConfigName is  the name of log4f config name.
```

The default value is not suitable to comment here.